from aiogram.utils.exceptions.base import TelegramAPIError


class ConflictError(TelegramAPIError):
    pass
